validation/pipeline/Jenkinsfile.recurring

```diff
@@ -224,65 +185,36 @@ node {
-                          string(name: 'TIMEOUT', value: timeout),
-                          text(name: 'CONFIG', value: configYaml),
-                          string(name: 'REPO', value: "${rancherRepo}"),
-                          string(name: 'BRANCH', value: "${branch}"),
-                          string(name: 'QASE_TEST_RUN_ID', value: "${testRunID}"),
-                          string(name: 'TEST_PACKAGE', value: "${env.TEST_PACKAGE}"),
-                          string(name: 'GOTEST_TESTCASE', value: "${runFlag} -p 1"),
-                          string(name: 'TAGS', value: tags),
-                        ]
+                    def goValidationParams = [
+                      string(name: 'TIMEOUT', value: timeout),
+                      text(name: 'CONFIG', value: configYaml),
+                      string(name: 'REPO', value: "${rancherRepo}"),
+                      string(name: 'BRANCH', value: "${branch}"),
+                      string(name: 'QASE_TEST_RUN_ID', value: "${testRunID}"),
+                      string(name: 'TEST_PACKAGE', value: "${env.TEST_PACKAGE}"),
+                      string(name: 'GOTEST_TESTCASE', value: "${env.GOTEST_TESTCASE_VALIDATION} -p 1"),
+                      string(name: 'TAGS', value: "${env.VALIDATION_TEST_TAG}"),
+                    ]
 
-                        jobs["$it-test-validation"] = { build job: individualJob, parameters: goValidationParams }
-                      } catch(err) {
-                        echo "Test Validation $it had failures: " + err
-                      }
-                    }
-
-                    if ("${env.TAGS}" != "") {
-                      sh "./yq e 'del(.provisioningInput.machinePools)' -i ${mainRancherConfigPath}"
-                      sh "./yq e 'del(.provisioningInput.nodePools)' -i ${mainRancherConfigPath}"
-                      def configYaml = readFile(file: "${mainRancherConfigPath}")
-
-                      def goValidationParams = [
-                        string(name: 'TIMEOUT', value: timeout),
-                        text(name: 'CONFIG', value: configYaml),
-                        string(name: 'REPO', value: "${rancherRepo}"),
-                        string(name: 'BRANCH', value: "${branch}"),
-                        string(name: 'QASE_TEST_RUN_ID', value: "${testRunID}"),
-                        string(name: 'TEST_PACKAGE', value: "${env.TEST_PACKAGE}"),
-                        string(name: 'TAGS', value: "${env.TAGS}"),
-                        string(name: 'GOTEST_TESTCASE', value: "${env.GOTEST_TESTCASE} -p 1"),
-                      ]
-
-                      jobs["provisioning-test-validation"] = { build job: individualJob, parameters: goValidationParams }
-                    
-                    }
+                    jobs["test-provisioning"] = { build job: individualJob, parameters: goProvisioningParams }
+                    jobs["test-validation"] = { build job: individualJob, parameters: goValidationParams }
```

Comment:
Probably should parameterize these job names in case they ever change for any reason.

Reply:
Hey Max I will change this, when I refactor the job with the new shared library stuff that Ugur has been working on. I have task in the backlog for it. For this go around I just wanted to be functional.